service/awsconfig/v10/resource/ebsvolume/delete.go

```diff
@@ -20,39 +20,38 @@ func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
 	// Get both the Etcd volume and any Persistent Volumes.
 	etcdVolume := true
 	persistentVolume := true
-
 	volumes, err := r.service.ListVolumes(customObject, etcdVolume, persistentVolume)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
 	if len(volumes) > 0 {
-		r.logger.LogCtx(ctx, "level", "info", "message", fmt.Sprintf("deleting %d EBS volumes", len(volumes)))
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("deleting %d EBS volumes", len(volumes)))
```

Comment:
Turning to debug since this kind of stuff is debug everywhere. Felt more clean to me.